src/configure/clients/github/TemplateServiceClient.ts

```diff
@@ -20,7 +19,7 @@ export class TemplateServiceClient {
             body);
     }
 
-    public async getTemplateParameters(templateId: string): Promise<ExtendedPipelineTemplate> {
+    public async getTemplateParameters(templateId: string): Promise<any> {
```

Comment:
Ideally we should have a fix return type, right? so, any specific reason for this change?

Reply:
+1

Reply:
While testing with default inputs I changed it, forgot to change back.